### ISSUE TITLE
YAML Headers And HTML Metadata Issues

### ISSUE THREAD

**Issue by tajmone** [jgm/pandoc#4086]
I'm facing some general issues with YAML headers in markdown documents.

For example, if I try to provide a specific string for `author-meta` (ie: have a different value than for `author`, to have different strings showing the document and its metadata) which contains an email address:

``` yaml
author-meta: John Doe <john.doe@dudemail.com>
```

... the email addressed get converted in a `mailto:` link:

``` html
<meta name="author" content="John Doe <a href="mailto:john.doe@dudemail.com">john.doe@dudemail.com</a>" />
```

If I try to escape the angle brackets:

``` yaml
author-meta: John Doe \<john.doe@dudemail.com\>
```
... , they get converted to HTML entities:

``` html
<meta name="author" content="John Doe &lt;john.doe@dudemail.com&gt;" />
```

These are the expected behaviours in markdown document, but pandoc should handle differently assignment of string to variable which concern HTML metadata or header stuff. Maybe the second case is correct, but chances are that the value of `author` meta key (unlike `description`) should allow angle brackets for machine parsing since it's not intended to be shown in browsers — but I'm not 100% sure.

Another problem I'm facing is trying to use double quotes within a `title` string — I've tried escaping the double quotes, enclosing the whole string in single quotes, etc., but to no avail. It might be just that I'm missing out some basic YAML rules here, but I have the impression that something changed here with pandoc v2 because in the past I managed to escape more or less any character I wanted to  see through.

**Comment by jgm** [comment]
If you really just want to set a template variable, don't
put it in metadata; use the `-V` option on the command line.

Or, if you want a metadata value that is not going to be
interpreted as Markdown, you can use the new raw inline syntax:

    author-meta: '`John Doe <john.doe@dudemail.com>`{=html}'

>   Another problem I'm facing is trying to use double quotes within a
>   title string

An example would be helpful.  YAML escaping rules can be a
bit complex, but it really depends on the particular case.
Often no escaping is needed, but if you start with a double
quote character, you may need single quotes round the whole
thing.

**Comment by tajmone** [comment]
Thanks a lot, great tip — I wasn't aware of the new raw inline syntax (still trying to get a grip on the tons of new features).

I'd like also to mention an issue I stumbled in regarding dates. If I set in YAML:

``` yaml
date: April 20, 2017
```

... it gets converted to `2017/05/2017` in the metadata date without any problem. But if the DAY field is a single-digit (eg: April 1, 2017) then it won't convert it to metadata. If my understanding is correct, that's part of teh Dublin standard which imposes the `YYYYMMDD` format; somehow pandoc seems to manage converting human-friendly dates but fails when day is a one digit --- looks like the whole thing could be fixed by appending a `0` to single-digit date-day elements.
